index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
 +-------------+------------+----------------------------------+-----------------+
 | **Version** | **Date**   | **Description**                  | **Owner**       |
 +=============+============+==================================+=================+
-| 0.1         | 3/06/2016  | Initial version.                 | Jacek Becla     |
+| 1.0         | 6/11/2016  | First version.                   | Jacek Becla     |
 +-------------+------------+----------------------------------+-----------------+
+
+
+
+.. _intro:
+
+Introduction
+============
+
+This document describes the project planning processes and tools used in the DM,
+with particular emphasis on software development.
+
+LSST is now in construction, funded through NSF MREFC. The planning process has
+to conform to all government standards and requirements, including mandated
+Risk and Earned Value Management System (EVMS).
```

Comment:
This reads badly, but I'm not sure what the correct parse is.

Is it a "mandated (Risk and Earned Value) Management System"?

Or "mandated Risk Management System and Earned Value Management System"?

Or something else?